lighthouse-core/audits/dobetterweb/uses-optimized-images.js

```diff
@@ -116,7 +115,9 @@ class UsesOptimizedImages extends Audit {
 
       totalWastedBytes += webpSavings.bytes;
       results.push({
-        url,
+        url: URL.getDisplayName(image.url),
+        previewUrl: image.url,
+        mimeType: image.mimeType,
```

Comment:
Yeah I think a comment like this was made on my 4th network record PR, after this was already done :) Though in this case we specifically needed to check mimetype in the gatherer to avoid GIF/SVG/etc